footprint/cortex-m/run_suite.py

```
@@ -0,0 +1,230 @@
+]
+TIMEOUT_RUN = 300  # seconds per QEMU run (4096-bit can take a while)
+TIMEOUT_BUILD = 600  # seconds for cargo build
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+TARGET_DIR = os.path.join(tempfile.gettempdir(), "rsa_footprint_cortexm")
+
+
+def run_cmd(args, timeout=TIMEOUT_RUN, **kwargs):
+    """Run a command, return (returncode, stdout, stderr)."""
+    env = os.environ.copy()
+    env.setdefault("CARGO_TARGET_DIR", TARGET_DIR)
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)